README.md

```diff
@@ -2,6 +2,26 @@
 
 SDK to be used to access Pasqal Cloud Services.
 
+## Version Support & Deprecation Timeline
+
+| Version | Release Date | End of Support | Notes                                       |
+| ------- | ------------ | -------------- | ------------------------------------------- |
+| 0.1.9   | Nov 2022     | Nov 2023       | [Changelog](CHANGELOG.md#016---2022-11-02)  |
```

Comment:
We can probably omit some of these entries? We would have officially deprecated them almost a year ago.

Maybe we should just consider listing things still in our support. Our audience is incredibly small and niche.

Reply:
Removed